python/examples/experimental/NLP.ipynb

```diff
@@ -0,0 +1,567 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "xUsdUYUbNrpl"
+      },
+      "source": [
+        ">### 🚩 *Create a free WhyLabs account to get more value out of whylogs!*<br> \n",
+        ">*Did you know you can store, visualize, and monitor whylogs profiles with the [WhyLabs Observability Platform](https://whylabs.ai/whylogs-free-signup?utm_source=whylogs-Github&utm_medium=whylogs-example&utm_campaign=String_Tracking)? Sign up for a [free WhyLabs account](https://whylabs.ai/whylogs-free-signup?utm_source=whylogs-Github&utm_medium=whylogs-example&utm_campaign=String_Tracking) to leverage the power of whylogs and WhyLabs together!*"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Y1-M8tfxNrpn"
+      },
+      "source": [
+        "# Natural Language Processing Logging\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "a6cXXyVONrpo"
+      },
+      "source": [
+        "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/whylabs/whylogs/blob/mainline/python/examples/advanced/String_Tracking.ipynb)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "WlasZRJiNrpo"
+      },
+      "source": [
+        "Blah blah blah NLP"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "PRGvUCCUNrpq"
+      },
+      "source": [
+        "## Installing whylogs\n",
+        "\n",
+        "If you haven't already, install whylogs: "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "pip install whylogs-1.1.7-py3-none-any.whl"
```

Comment:
remove the pip install referencing whylogs wheel, you have a working pip install in next cell.